pp_netlib/functions.py

```diff
@@ -1,6 +1,11 @@
 ########################
 ####   .CONSTRUCT   ####
 ########################
+from ast import Index
+from collections import defaultdict, Counter
+from functools import partial
+from multiprocessing import Pool, freeze_support
```

Comment:
I don't think `freeze_support` is used?

Reply:
multiprocessing is not used for now, removing this